lua/clipboard-image/config.lua

```diff
@@ -57,12 +61,17 @@ end
 ---@param config_toload table
 ---@return table loaded_config
 M.load_config = function(config_toload)
+  local img_format = M.load_opt(config_toload.img_format)
+  local img_ext = "."..img_format
+  if img_format == "jpg" then img_format = "jpeg" end
   return {
     affix = M.load_opt(config_toload.affix),
     img_name = M.load_opt(config_toload.img_name),
     img_dir = M.load_opt(config_toload.img_dir),
     img_dir_txt = M.load_opt(config_toload.img_dir_txt),
     img_handler = config_toload.img_handler,
+    img_format = img_format,
+    img_ext = img_ext,
```

Comment:
The reason I added both `img_format` and `img_ext` is to deal with JPEG images. Some people might prefer to use the extension `jpg` rather than `jpeg`. However, the clipboard facility:

```
"xclip -selection clipboard -t image/%s -o > '%s'"
```

will only understand `jpeg` and not `jpg`.

Reply:
That's a good idea. However, I'm not sure how to resolve mimetypes. I tried working with mimetypes on an unrelated problem and didn't quite get a satisfactory solution :cry: .

Feel free to upload your own suggestion on top of this PR if you have an idea.

Reply:
Ok looking forward to it.